pkg/verify/configmap.go

```diff
@@ -0,0 +1,87 @@
+package verify
```

Comment:
what are those `.un~` files above?

Reply:
My mistake. Yes, I need to remove these backup related files. I'm putting this on hold until I have other significant changes pushed as well. Thanks for reviewing.